packages/cli/src/commands/logout/index.ts

```diff
@@ -13,8 +13,11 @@ import { logoutCommand } from './command';
 import { getFlagsSpecification } from '../../util/get-flags-specification';
 import output from '../../output-manager';
 import { LogoutTelemetryClient } from '../../util/telemetry/commands/logout';
+import { logout as future } from './future';
+
+export default async function logout(client: Client): Promise<number> {
+  if (client.argv.slice(2).includes('--future')) return await future(client);
```

Comment:
If this is shifted down, `--future` will be available as `parsedArgs.flags['--future']`